tests/docker_run_tests.rs

```diff
@@ -270,6 +270,27 @@ fn test_pnpm() {
     assert!(output.contains("Hello from PNPM"));
 }
 
+#[test]
+fn test_bun() {
+    let name = simple_build("./examples/node-bun");
+    let output = run_image(name, None);
+    assert!(output.contains("Hello from Bun"));
+}
+
+#[test]
+fn test_bun_no_start() {
+    let name = simple_build("./examples/node-bun-no-start");
+    let output = run_image(name, None);
+    assert!(output.contains("Hello from Bun"));
+}
```

Comment:
I think we only need one of these docker tests since the generate plan test will cover the other case